go.mod

```diff
@@ -354,3 +354,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+retract [v0.1.0, v1.14.9]
```

Comment:
Do we want to retract priror API & SDK versions as well, in the respective modules?

Reply:
I will retract those as well.